src/__tests__/App.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import store from '../redux/store';
+import App from '../App';
+
+describe('App component', () => {
+  test('renders', () => {
+    const tree = render(
+      <Provider store={store}>
+        <App />
+      </Provider>,
```

Comment:
- [ ] All tests for the app are failing, Please make sure that all tests are passing without any error.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/170495031-5b8c7934-9a54-4fd6-b83b-34df79c6e9cd.png)